### PR TITLE
[DC-1764] Add CT/RT clean validation notebook.

### PR DESCRIPTION
**Pull request by ksdkalluri**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+# # Simplified Clean Height Check
+# Not checking all the clean height bells and whistles.  Just checking to make sure no site
+# has submitted rows with measurement_concept_id in (3036277, 3023540, 3019171).  If any
+# of these values are greater than 0, a portion of the clean_height_weight rule failed.
+# Based on query defined in line 249 of clean_height_weight.py.
+# # IGNORE THESE RESULTS.  BAD ANALYSIS. 
+
+query = f'''
+SELECT 
+COUNTIF(m.measurement_concept_id = 3036277) as n_3036277,
```
Comment (lrwb-aou) [inline data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py:262]: @ksdkalluri I would recommend getting rid of these two queries entirely.  They are dead code and do not function as intended.  They are also blatantly skipped.

Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py]: Done Updated!

File: data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+     'measurement' as table_to_drop_from
+    FROM `{project_id}.{clean_dataset}.measurement`
+
+#    UNION ALL
+#
+#    SELECT
+#     COUNTIF((cause_source_concept_id is null or cause_source_concept_id = 0) AND
+#             (cause_concept_id is null or cause_concept_id = 0)) as n_records_to_drop,
+#     'death' as table_to_drop_from
+#    FROM `{project_id}.{clean_dataset}.death`
```
Comment (lrwb-aou) [inline data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py:222]: @ksdkalluri go ahead and remove this code.  it is essentially dead.

File: data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+# Set all value_as_number = 9999999 to NULL.  If not null, an error happened.
+
+query = f'''
+SELECT 
+ COUNTIF(value_as_number = 9999999) as n_not_changed,
+ CASE WHEN COUNTIF(value_as_number = 9999999) > 0 THEN 'FAILURE'
+  ELSE 'SUCCESSFUL'
+ END as check_passed
+FROM `{project_id}.{clean_dataset}.measurement`
+'''
```
Comment (lrwb-aou) [inline data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py:55]: how do you feel about changing these sql statements to using the `JINJA_ENV` formatting?  It is safer than using f-strings to generate the queries.

Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py]: I agree, updated.

File: data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py (hunk 2 above)
Comment (ksdkalluri) [reply data_steward/analytics/cdr_ops/ad_hoc_analyses/CT_RT_clean_ckecks.py]: death is now dead.

**Review by lrwb-aou — APPROVED** [review]
Looks good.  Everything that can pass does.

**Pull request merged**